docs/self-hosted/config-options.md

```diff
@@ -276,6 +276,22 @@ All the `DB_POOL__` prefixed options are passed to [`tarn.js`](https://github.co
 
 :::
 
+## Redis
+
+Directus requires Redis for multi-container deployments. This ensures that things like caching, rate-limiting, and
+WebSockets work reliably across multiple containers of Directus.
+
+| Variable         | Description                                                                                                                                                | Default Value |
+| ---------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------- |
+| `REDIS`          | Redis connection string, e.g., `redis://user:password@127.0.0.1:6380/4`. Using this will ignore the other Redis connection parameter environment variables | --            |
+| `REDIS_HOST`     | Hostname of the Redis instance, e.g., `"127.0.0.1"`                                                                                                        | --            |
+| `REDIS_PORT`     | Port of the Redis instance, e.g., `6379`                                                                                                                   | --            |
+| `REDIS_USERNAME` | Username for your Redis instance, e.g., `"default"`                                                                                                        | --            |
+| `REDIS_PASSWORD` | Password for your Redis instance, e.g., `"yourRedisPassword"`                                                                                              | --            |
+
+If the Redis configuration is omitted, Directus will default to using local memory. This will fail when running Directus
+across multiple containers.
```

Comment:
Should probably be slightly rephrased to something like "Redis is required when running Directus across multiple containers"